### ISSUE TITLE
Server side subscriptions are not triggered

### ISSUE THREAD

**Issue by marktani** [graphcool/prisma#2169]
# Bug Report

**Current behavior**

Server side subscriptions are not triggered.

**Reproduction**

```
npm install -g prisma
prisma init a
cd a
prisma deploy
```

Add `subscriptions` to `prisma.yml` file:

```yml
subscriptions:
  userChangedEmail:
    webhook:
      url: https://hookbin.com/bin/vgLJlpDW # REPLACE with your own webhook
      headers:
        Content-Type: application/json
        Authorization: Bearer cha2eiheiphesash3shoofo7eceexaequeebuyaequ1reishiujuu6weisao7ohc
    query: |
      subscription {
        user {
          node {
            name
          }
        }
      }
```

```sh
prisma deploy
prisma playground
```

```graphql
mutation {
  createUser(data: {
    name: "Nilan"
  }) {
    id
  }
}
```

Refresh browser at webhook URL. No event is triggered.

**Expected behavior?**

Event is triggered.

**Comment by marktani** [comment]
This was fixed in #2171, it is available in `1.5.2`.

**Comment by everdrone** [comment]
Experiencing this in v1.6.3 with "database only" template
`prisma/1.6.3 (darwin-x64) node-v9.5.0`

config
```
service: test-sss

stage: dev

secret: mysecret123

datamodel: datamodel.graphql

cluster: local


# server side subscriptions
subscriptions:
  sendVerificationEmailForSignup:
    webhook:
      url: http://requestbin.fullcontact.com/1nmaoqq1
      headers:
        Content-Type: application/json
        Authorization: Bearer myCoolTokenLolz
    query: |
      subscription {
        user {
          node {
            id
            email
            firstName
          }
        }
      }
```
